resources/qml/dialogs/ReadReceipts.qml

```diff
@@ -58,59 +58,54 @@ ApplicationWindow {
                 boundsBehavior: Flickable.StopAtBounds
                 model: readReceipts
 
-                delegate: RowLayout {
-                    spacing: Nheko.paddingMedium
-
-                    Avatar {
-                        width: Nheko.avatarSize
-                        height: Nheko.avatarSize
-                        userid: model.mxid
-                        url: model.avatarUrl.replace("mxc://", "image://MxcImage/")
-                        displayName: model.displayName
-                        onClicked: room.openUserProfile(model.mxid)
-                        ToolTip.visible: avatarHover.hovered
-                        ToolTip.text: model.mxid
-
-                        HoverHandler {
-                            id: avatarHover
-                        }
-
+                delegate: ItemDelegate {
+                    onClicked: room.openUserProfile(model.mxid)
+                    padding: Nheko.paddingMedium
+                    width: receiptLayout.implicitWidth
+                    height: receiptLayout.implicitHeight
+                    background: readReceiptsRoot.color
```

Comment:
This should be an Item, right? Either use an empty Item{} or a rectangle with that color (no need to set a height)

Reply:
Hmm, I think you're right.